next.config.js

```diff
@@ -120,6 +120,7 @@ module.exports = withPlugins([[withBundleAnalyzer]], {
     VERCEL_URL: process.env.VERCEL_URL,
     SITE_IMAGERY_API_URL: SITE_IMAGERY_API_URL,
     WIDGET_URL: process.env.WIDGET_URL,
+    CONFIG_URL: process.env.CONFIG_URL,
```

Comment:
Please also put examples of CONFIG_URL now in our .env files in the repository:
- [.env.development](https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.development)
- [.env.example](https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.example)
- [.env.local.example](https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.local.example)
- [.env.production](https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.production)